store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -339,6 +339,28 @@ public CarbonWriter buildWriterForCSVInput(Schema schema)
     return new CSVCarbonWriter(loadModel);
   }
 
+  /**
+   * Build a {@link CarbonWriter}, which accepts row in CSV format
+   * @param schema carbon Schema object {org.apache.carbondata.sdk.file.Schema}
+   * @param numOfThreads number of threads() in which .write will be called.
+   * @return CSVCarbonWriter
+   * @throws IOException
+   * @throws InvalidLoadOptionException
+   */
+  public CarbonWriter buildWriterForCSVInput(Schema schema, short numOfThreads)
```

Comment:
Better name as `buildThreadSafeWriterForCSVInput` and update java doc like how this writer can be used in multithreaded environment